pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -1455,6 +1455,7 @@ public void getDeduplicationEnabled(@Suspended final AsyncResponse asyncResponse
                              @PathParam("namespace") String namespace,
                              @PathParam("topic") @Encoded String encodedTopic) {
         validateTopicName(tenant, namespace, encodedTopic);
+        validateTopicOwnership(topicName, true);
```

Comment:
we don't need to validate the topic owner ship for topic level policy since after the policy updated, the broker will get a policy change event, if the broker own the topic, the broker will apply the policy change to that topic